modules/end-user-guide/pages/url-parameter-for-workspace-image.adoc

```diff
@@ -0,0 +1,14 @@
+:_content-type: CONCEPT
+:description: URL parameter for the workspace image
+:keywords: parameter, URL, workspace, image
+:navtitle: URL parameter for the workspace image
+//:page-aliases: url-parameter-for-workspace-image.adoc
+
+[id="url-parameter-for-workspace-image"]
+= URL parameter for the workspace image
+
+You can use the `image` parameter to add a path to a container image in the following scenarios:
+
+* When the Git repository contains no devfile, and you want to start a new workspace with this container.
+
+* When the Git repository contains a devfile, and you want this container image to run instead of the first `container` `image` listed in the `components` section of the devfile.
```

Comment:
As I look at these two bullet points, the leading `When` seems to be redundant in combination with `, and`, although I completely understand where you're coming from. To me, these sentences are easier to process mentally as complete sentences without `When`, hence this suggestion:
```suggestion
* The Git repository contains no devfile, and you want to start a new workspace with this container.

* The Git repository contains a devfile, and you want this container image to run instead of the first `container` `image` listed in the `components` section of the devfile.
```